libr/util/print.c

```diff
@@ -1629,6 +1631,26 @@ static bool issymbol(char c) {
 	}
 }
 
+static bool check_arg_name (char *p, RList *vars) {
+	bool found_var = false;
+	if (vars) {
+		int z;
+		for (z = 0; p[z] && (IS_ALPHA (p[z]) || IS_DIGIT (p[z]) || p[z] == '_'); z++);
+		char tmp = p[z];
+		p[z] = '\0';
+		RAnalVar *var;
+		RListIter *iter;
+		r_list_foreach (vars, iter, var) {
```

Comment:
I think instead of iterating through list of vars , us this function  `r_anal_var_get_byname` , it's faster since it uses sdb query to retrieve the var , also by doing this way we can avoid this hack and drop `RList *vars` from `RPrint`

Reply:
Hey this is a great idea, it solves a lot of problems!
Now I'm going to sleep, tomorrow I'll modify it for sure.
Thanks :)